lib/extensions/extensions.js

```diff
@@ -0,0 +1,100 @@
+// // Imports
```

Comment:
It feels that this file could be more readable. Would it be possible to give at the beginning of the file a high level view, maybe in steps, of what this file does? 🙂

Reply:
Could do, do you have an example of that kind of comment?

Reply:
I was thinking something like [this](https://github.com/lodash/lodash/blob/4ea8c2ec249be046a0f4ae32539d652194caf74f/countBy.js#L7) or [this](https://github.com/alphagov/govuk-frontend/blob/master/src/components/accordion/accordion.js#L2). 

But @aliuk2012 and I ended up writing out [a short description and listing out the exports](https://github.com/alphagov/govuk-prototype-kit/blob/48613ff75d026e6619181cef94e63e267f23aae7/lib/extensions/extensions.js#L1) which is good for now.